py_nltk/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM socrata/py_analysis
+
+RUN pip install nltk==3.0.0
+
+RUN mkdir -p /home/socrata
+
+ENV HOME /home/socrata
+
+RUN python -m nltk.downloader -d $HOME/nltk_data stopwords &&\
+    python -m nltk.downloader -d $HOME/nltk_data punkt &&\
+    python -m nltk.downloader -d $HOME/nltk_data wordnet &&\
+    python -m nltk.downloader -d $HOME/nltk_data maxent_treebank_pos_tagger
```

Comment:
This is limited for an nltk import with the downloader. Why are only these data packs installed? Why not fewer or all?